ReactNative/ReactTwitter/core/views/debug/debug-native-component-scene.js

```diff
@@ -43,7 +45,54 @@ var DebugNativeComponentScene = React.createClass({
   },
 
   _renderRow (rowData: string) {
-    return <Text style={styles.text}>{rowData}</Text>
+    let enabled = this._enabledForRowData(rowData)
+    let textColor = this._colorForRowData(rowData)
+    let textSize = this._sizeForRowData(rowData)
```

Comment:
They are not quite styles, but props, but I will take a look into a way of grouping them.